app/layout.tsx

```diff
@@ -1,6 +1,7 @@
 import type { Metadata } from "next";
 import { Inter } from "next/font/google";
-import "./globals.css";
+import "@/fonts/globals.css";
+import Banner from "@/components/ui/Banner";
```

Comment:
The path should be @/components/Banner. Forget about 'ui'. It will occur only if we use Shadcn.